docs/using_python_client_with_hazelcast_imdg.rst

```diff
@@ -1349,19 +1349,182 @@ the entry processor in the ``Map`` methods. See the following example.
+Hazelcast uses these mappings to access resources in these data sources as if
+they were SQL tables.
+
+Hazelcast provides `connectors
+<https://docs.hazelcast.com/hazelcast/latest/sql/connectors.html>`__ that allow
+you to create mappings to Apache Kafka, your local file system, and distributed
+maps. Using these mappings, you can run SQL queries against data in these data
+sources.
+
+For example, this query creates a mapping to a Kafka topic that streams trading
+data as JSON:
+
+.. code:: sql
+
+    CREATE MAPPING trades(
+        __key BIGINT,
+        ticker VARCHAR,
+        amount INT)
+    TYPE Kafka
+    OPTIONS (
+        'keyFormat' = 'bigint',
+        'valueFormat' = 'json',
+        'bootstrap.servers' = '127.0.0.1:9092');
+
+The following query gets the full company name for each ticker from a Hazelcast
+map called ``companies``.
+
+.. code:: sql
+
+    SELECT
+        trades.ticker, trades.amount, companies.name
+    FROM
+        trades
+    JOIN companies
+    ON trades.ticker = companies.ticker;
+
+SQL Statements
+~~~~~~~~~~~~~~
+
+**Data Manipulation Language(DML) Statements**
+
+- `SELECT <https://docs.hazelcast.com/hazelcast/latest/sql/select.html>`__:
+  Read data from a table.
+- `SINK INTO/INSERT INTO
+  <https://docs.hazelcast.com/hazelcast/latest/sql/sink-into.html>`__:
+  Ingest data into a map and/or forward data to other systems.
+- `UPDATE <https://docs.hazelcast.com/hazelcast/latest/sql/update.html>`__:
+  Overwrite values in map entries.
+- `DELETE <https://docs.hazelcast.com/hazelcast/latest/sql/delete.html>`__:
+  Delete map entries.
+
+**Data Definition Language(DDL) Statements**
+
+- `CREATE MAPPING
+  <https://docs.hazelcast.com/hazelcast/latest/sql/create-mapping.html>`__:
+  Map a local or remote data object to a table that Hazelcast can access.
+- `SHOW MAPPINGS
+  <https://docs.hazelcast.com/hazelcast/latest/sql/show-mappings.html>`__:
+  Get the names of existing mappings.
+- `DROP MAPPING
+  <https://docs.hazelcast.com/hazelcast/latest/sql/drop-mapping.html>`__:
+  Remove a mapping.
+
+**Job Management Statements**
+
+- `CREATE JOB
+  <https://docs.hazelcast.com/hazelcast/latest/sql/create-job.html>`__:
+  Create a job that is not tied to the client session.
+- `ALTER JOB
+  <https://docs.hazelcast.com/hazelcast/latest/sql/alter-job.html>`__:
+  Restart, suspend, or resume a job.
+- `SHOW JOBS
+  <https://docs.hazelcast.com/hazelcast/latest/sql/show-jobs.html>`__:
+  Get the names of all running jobs.
+- `DROP JOB <https://docs.hazelcast.com/hazelcast/latest/sql/drop-job.html>`__:
+  Cancel a job.
+- `CREATE OR REPLACE SNAPSHOT (Enterprise only)
+  <https://docs.hazelcast.com/hazelcast/latest/sql/create-snapshot.html>`__:
+  Create a snapshot of a running job, so you can stop and restart it at a
+  later date.
+- `DROP SNAPSHOT (Enterprise only)
+  <https://docs.hazelcast.com/hazelcast/latest/sql/drop-snapshot.html>`__:
+  Cancel a running job.
+
```

Comment:
Can we add the information in https://docs.hazelcast.com/hazelcast/5.0-snapshot/query/sql-overview.html#querying-existing-maps to here without duplicating information?

Reply:
No, the info here is already in the next section. I don't see a point in doing that